src/plugins/intel_gpu/src/kernel_selector/cl_kernels/dynamic_quantize_gpu_opt.cl

```diff
@@ -8,6 +8,14 @@
 #error "dynamic_quantize_gpu_opt.cl: Unsupported output dimension"
 #endif
 
+#define VLOAD_N CAT(vload, VEC_SIZE)
+#define VSTORE_N CAT(vstore, VEC_SIZE)
+#define HALF_N CAT(half, VEC_SIZE)
+#define CONVERT_CHAR_N CAT(convert_char, VEC_SIZE)
+#define AS_TYPE_N_(type, n, x) as_##type##n(x)
+#define AS_TYPE_N(type, n, x) AS_TYPE_N_(half, n, x)
```

Comment:
why is it fixed as `half`?

Reply:
Original code was implemented as half8. I thought the target was half. Do we need to consider all format? I will fix it by coming commit.

Reply:
Fixed to INPUT0_TYPE